editor/editor_node.cpp

```diff
@@ -3973,8 +3973,8 @@ void EditorNode::_dock_make_float() {
 	Control *dock = dock_slot[dock_popup_selected]->get_current_tab_control();
 	ERR_FAIL_COND(!dock);
 
-	Size2 dock_size = dock->get_size(); //remember size
-	Point2 dock_screen_pos = dock->get_global_position() + get_tree()->get_root()->get_position();
+	Size2 dock_size = dock->get_size() + Size2i(8, 8) * EDSCALE; //remember size
```

Comment:
Maybe this can be made a (local) constant, to make it clear that we're using the same value for the size/pos here and the margins below?